### DISCUSSION TITLE
Crash on ios after update lib to 8.0.2

### DISCUSSION THREAD

**Discussion by ginger-777** [MvvmCross/MvvmCross#4265] [Q&A]
We have faced an issue after we updated MvvmCross from the 7 to 8 version.

In debug mode or in dev configuration on device and simulator everything is ok.
But if creating a build for distribution to testfligth app is crashed on the splash screen and no crash log we can catch.

Can anybody help with this?

**Comment by stefan89** [comment]
I have the exact same issue. Any update on this?

**Reply by Cheesebaron** [reply]
Not much to go off here. Add some diagnostics to your app and check device log.

**Reply by stefan89** [reply]
@Cheesebaron when I add 

`<MtouchDebug>true</MtouchDebug> `

to the configuration, it does not crash anymore. This is not desired for a release build. Any suggestions for this?

**Reply by ginger-777** [reply]
We have fixed this issue by updating MvvmCross.Plugin.Messenger and  MvvmCross.Plugin.Color to 8.0.**1**, rest of MvvmCross libs are 8.0.**2** version
Also, double-check that all MvvmCross package references contains "Culture=neutral, processorArchitecture=MSIL".

**Reply by stefan89** [reply]
Thanks @ginger-777. Solved the issue for me as well.

**Reply by zoooN** [reply]
Faced with the same issue today. Downgrading MvvmCross.Plugin.Messenger to 8.0.1 helped. @ginger-777 Thanks
